src/topics/unread.js

```diff
@@ -115,10 +117,28 @@ module.exports = function (Topics) {
 		});
 
 		const unreadTopics = _.unionWith(categoryTids, followedTids, (a, b) => a.value === b.value)
-			.filter(t => !ignoredTids.includes(t.value) && (!userReadTime[t.value] || t.score > userReadTime[t.value]))
+			.filter(filterAlreadyRead)
 			.concat(tids_unread.filter(t => !ignoredTids.includes(t.value)))
 			.sort((a, b) => b.score - a.score);
 
+		function filterAlreadyRead(topic) {
+			if (ignoredTids.includes(topic.value)) {
+				return false;
+			}
+
+			const userReadTime = userReadTimes[topic.value];
+			if (!userReadTime) {
+				return true;
+			}
+
+			const isMainPost = parseInt(topicPostCounts[topic.value], 10) === 1;
```

Comment:
I don't understand what this is doing. Why are we removing topics that only have 1 post? There might be a better way of doing this without loading all the topics postcounts.

Reply:
:point_up: Same.